app/permission/rest.py

```diff
@@ -0,0 +1,54 @@
+
+from flask import (jsonify, request, abort, Blueprint, current_app)
+from app.schemas import permission_schema
+from app.errors import register_errors
+from app.dao.permissions_dao import permission_dao
+
+permission = Blueprint('permission', __name__)
+register_errors(permission)
+
+
+@permission.route('', methods=['GET'])
```

Comment:
Don't think we need a rest endpoint for permissions - we'd only operate on them via user actions.